### ISSUE TITLE
> is replaced with &gt; in the Inner Table with boxable

### ISSUE THREAD

**Issue by mailidpankaj** [dhorions/boxable#216]
Hi,

I am trying to write below string in the Inner Table using the boxable library with PDFBox to write to PDF:

Composers->Schubert->String

but, the it gets printed in PDF as below:

Composers-"&gt"Schubert-"&gt"String

I investigated on the HTML tags and found that we can use Entity Numbers in place of > sign, I tried below values:

- replaced > with &#62  - still it prints &gt in the PDF

where when I replace > with &#63; or &#64; , I can see the special characters like @ and ? gets printed in the PDF.

Hence, only in case of &#62; nothing changes and > doesn't get printed in PDF.

Looks like there is a limitation with boxable inner table API createTableCell().

Please let me know how can I print the ">" character in the inner table.

Regards,
Pankaj

**Comment by CSchemmy** [comment]
I have see at sample 8 a ">". I have change line 775 of TableTest to` 		cell = additionArow.createCell(20f, " Test->Test1 ");` and it works.

**Comment by CSchemmy** [comment]
I have found the problem. The Problem can be solved at code TableCell at line 173 and 176. 

`			for (Element col : tableHasHeaderColumns ? tableHeaderCols : tableCols) {
				if (col.attr("colspan") != null && !col.attr("colspan").isEmpty()) {
					Cell<T> cell = (Cell<T>) row.createCell(
							tableWidth / columnsSize * Integer.parseInt(col.attr("colspan")) / row.getWidth() * 100,
							col.text());
				} else {
					Cell<T> cell = (Cell<T>) row.createCell(tableWidth / columnsSize / row.getWidth() * 100,
							col.text());
				}
`

**Comment by mailidpankaj** [comment]
Hi,

Thanks for fixing this issue. I need to access this fix. Did you merge the fix to "dhorions/boxable" ? 

I checked the TableCell.java file and couldn't find the fix. Please let me know how can I access the fix? Which version of boxable library do I need to get this fix?

Regards,
Pankaj

**Comment by CSchemmy** [comment]
I have no rights to merge them. A pull request is still open #224

**Comment by mailidpankaj** [comment]
Ok. Any idea how can I access this fix in my application? I am using "dhorions/boxable" library.

**Comment by CSchemmy** [comment]
Yes, you need to pull a private copy and change the version.  This can be done with the following script

```shell
git clone https://github.com/dhorions/boxable.git
cd boxable/
mvn build-helper:parse-version versions:set -DnewVersion=boxable-1.6-my-SNAPSHOT versions:commit
mvn clean install -Dgpg.skip=true
```
Then you can change the version in your project from boxable-1.6 to boxable-1.6-my-SNAPSHOT. Take a name with the extension SNAPSHOT so you can build the library multiple times.

After you have implemented the changes from the pull request #224 you have to rebuild the library.

```
mvn clean install -Dgpg.skip=true
```

Then you can test your project. 

If pull request #224 is merged you can go back to the merged version from the library.

**Comment by CSchemmy** [comment]
Note that <text> is interpreted as HTML tag. Furthermore, the character "-" is also a character for the line break within a cell.
